hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -516,100 +525,113 @@ async def _create_order(self,
                 "respInst": "ACCEPT",
             }
             self.start_tracking_order(
-                order_id,
-                None,
-                trading_pair,
-                trade_type,
-                price,
-                amount,
-                order_type
+                order_id=order_id,
+                exchange_order_id=ascend_ex_utils.gen_exchange_order_id(self._account_uid, order_uuid, timestamp)[0],
```

Comment:
So I was reading AscendEx API documentation and they did provide us with a how they generate the `exchange_order_id`. I felt that it would be beneficial to use it since that way we do not need to really wait for a confirmation to associate the `client_order_id` with an `exchange_order_id`.